src/http_rest_api_handler.cpp

```diff
@@ -84,7 +84,10 @@ Status HttpRestApiHandler::dispatchToProcessor(
             request_components.model_version_label, response);
     }
     if (request_components.type == ConfigReload) {
-        return processModelControlApiRequest(*response);
+        return processConfigReloadRequest(*response);
+    }
+    if (request_components.type == ConfigStatus) {
+        return processConfigStatusRequest(*response);
     }
     return StatusCode::UNKNOWN_ERROR;
```

Comment:
Minor and does not to be fixed here but in such cases error is known -> UNKNOWN_REQUEST_COMPONENTS_TYPE or sth similar would make debug easier in case of encountering such issue.